modules/apps/object/object-rest-impl/rest-openapi.yaml

```diff
@@ -460,6 +460,31 @@ paths:
                             schema:
                                 $ref: "#/components/schemas/ObjectEntry"
             tags: ["ObjectEntry"]
+    "/by-external-reference-code/{externalReferenceCode}/by-version/{version}/copy":
+        post:
+            operationId: postByExternalReferenceCodeByVersionCopy
+            parameters:
```

Comment:
Hey @MarinhoFeliphe,

I'm estimating M2, and I noticed that the team will need to implement this story: [Allow Users to Create a Copy of an Existing Version](https://liferay.atlassian.net/browse/LPD-44734).

I believe we should use the term `restore` to implement this current story, as Alejandro previously proposed in this [thread](https://liferay.slack.com/archives/C03S91MJG6L/p1741177932526349?thread_ts=1740774603.819079&cid=C03S91MJG6L).

What do you think?

Reply:
Makes sense to me